Api/Data/StateDataInterface.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Api\Data;
+
+interface StateDataInterface
+{
+    /**
+     * Constants for keys of data array. Identical to the name of the getter in snake case.
+     */
+
+    /*
+     * Entity ID.
+     */
+    const ENTITY_ID = 'entity_id';
+
+    /*
+     * Quote ID.
+     */
+    const QUOTE_ID = 'quote_id';
+
+    /*
+     * Payment State Data.
+     */
+    const STATE_DATA = 'state_data';
```

Comment:
shouldn't we provide some more information in the phpdocs (so it will also show up in the swagger doc) like `stringified json returned by the adyen checkout component`.

Reply:
I've added a description for the function and the params, but only the function one shows up in Swagger. Do you know if this is done differently for the params?